admin/config/feature-toggles.default.json

```diff
@@ -7,6 +7,6 @@
   "pupilUpload": true,
   "dependencyTracking": true,
   "accessArrangements": false,
-  "prepareCheckMessaging": false,
+  "prepareCheckMessaging": true,
```

Comment:
@jon-shipley why changing the default value to true at this stage?

Reply:
@GeorgeChatzigiannis - it shouldn't be changed yet.